### PR TITLE
TeacherconRegistration eyes test. Keep this separate for improved spe…

### PR DESCRIPTION
**Pull request by mehalshah**
…ed when testing

### CODE REVIEW

File: dashboard/test/ui/features/pd/teacherconRegistration.feature [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  # Section 1
+  And I wait until element "h4" contains text "Section 1: Are you joining us?"
+  And I press the first "input[name='teacherAcceptSeat']" element
+  Then I see no difference for "Section 1: Are you joining us?"
+  And I press the first "button#next" element
+
+  # Section 2
+  And I wait until element "h4" contains text "Section 2: Travel Plans"
+  And I wait for 5 seconds
```
Comment (Hamms) [inline dashboard/test/ui/features/pd/teacherconRegistration.feature]: why these 5-second waits?

File: dashboard/test/ui/features/pd/teacherconRegistration.feature [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  And I press the first "#agreeShareContact" element
+  Then I see no difference for "Section 4: Releases"
+  And I submit
+
+  # Confirmation
+  Then I wait until element "h1" contains text "Thank you for registering for TeacherCon!"
+  Then I see no difference for "TeacherCon confirmation form"
+  Then I close my eyes
+
+
```
Comment (Hamms) [inline dashboard/test/ui/features/pd/teacherconRegistration.feature]: nit: extra whitespace

File: dashboard/test/ui/features/pd/teacherconRegistration.feature (hunk 1 above)
Comment (mehalshah) [reply dashboard/test/ui/features/pd/teacherconRegistration.feature]: Without it, I seem to trigger
```
And I press the first "input[name='howOfferCsp']" element                                           # features/step_definitions/steps.rb:267
[pd/teacherconRegistration]       unknown error: Element <input type="radio" value="As an AP course" label="As an AP course" name="howOfferCsp"> is not clickable at point (53, 6). Other element would receive the click: <input type="radio" value="As a non-AP course" label="As a non-AP course" name="howOfferCsp">
```

Comment (Hamms) [reply dashboard/test/ui/features/pd/teacherconRegistration.feature]: Oh, funky

**Pull request merged**